### PR TITLE
feat(customers): add customer create endpoint

### PR DESCRIPTION
**Pull request by DewaldoDev**
### Problem
In order to use solutions like Customer Voice or Inbox, we need the ability to sync contact information from an SMB's system (be it CRM or POS) onto our platform.

### Solution
Add an endpoint for exposing the `contacts` µservice Customers API allowing an SMB to create a customer record.

@richard-rance @wfehr-va @Short-Hop @ncline-va @hchow-va 
[CUS-1859]

[CUS-1859]: https://vendasta.jira.com/browse/CUS-1859?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by Short-Hop — APPROVED** [review]

File: docs/Guides/CreateCustomer.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
+---
+tags: [Customers]
+---
+# Sync Contact Information From Your System to Vendasta
+
+> This guide relies on "Proposed" endpoints and is likely to change until all features have been made generally available.
+
+Vendasta allows you to sync contact information from your CRM or POS system onto our platform. Doing so will allow you to use features in our platform like SMS messaging through Vendasta's Inbox or [requesting business review through Customer Voice](./CustomerVoice/SendReviewRequest.md).
```
Comment (richard-rance) [inline docs/Guides/CreateCustomer.md:8]: We should address the difference between sales contacts for partners and customers for SMBs. (And clean up the terminology)

Comment (DewaldoDev) [reply docs/Guides/CreateCustomer.md:8]: Yeah, that's an issue. Are we safe sticking with the model name "Customers" or should we opt for a more descriptive one?

File: toc.json [hunk 2]
```diff
@@ -100,6 +100,11 @@
           "type": "item",
           "title": "Displaying User Info",
           "uri": "docs/Guides/Display-Users.md"
+        },
```
Comment (richard-rance) [inline toc.json:103]: The new business.yaml will need to be added as well.

File: openapi/business/business.yaml [hunk 3]
```diff
@@ -0,0 +1,321 @@
...
+    description: Production
+  - url: 'https://demo.apigateway.co/business'
+    description: Demo
+  - url: '{local}/products/reviews'
+    description: Local
+  - url: 'http://localhost:11001/business'
+    description: Localhost
+paths:
+  /customers:
+    post:
```
Comment (richard-rance) [inline openapi/business/business.yaml:17]: You will need to define the `option` method as well.

Comment (DewaldoDev) [reply openapi/business/business.yaml:17]: Yep, it's there, just further down.

**Pull request merged**